wiki/Client/Interface/de.md

```diff
@@ -1,203 +1,325 @@
+  - Unter den öffentlichen Ranglisten (z. B. Global, Friends, usw.) wird dein höchster Score unten dargestellt sowie dein Rang auf der Rangliste.
+- Wenn `Kein Rekord vorhanden` erscheint, bedeutet es, dass noch keine Replays für die aktuelle Ansichtseinstellung vorhanden ist (das ist meistens in der lokalen Ansichtseinstellung zu sehen, wenn du gerade eine Beatmap heruntergeladen oder editiert hast).
+
+Dies sind die Ansichtseinstellungen:
+
+- Lokale Rangliste
+- Nationale Rangliste\*
+- Globale Rangliste
+- Globale Rangliste (Aktive Mods)\*
+- Freundesrangliste\*
+
+\*Benötigt [osu!supporter](/wiki/osu!supporter).
+
+Klicke auf das Sprechblasensymbol, um einen **Schnellzugang zu dieser Beatmap im Web** für die ausgewählte Beatmap zu erhalten:
+
+- Drücke `1` oder klicke auf die Fläche `Beatmap-Übersicht/Rekorde`, um die Beatmapseite für den ausgewählten Schwierigkeitsgrad in deinem Browser zu sehen.
+- Drücke `2` oder klicke auf die Fläche `Beatmap-Thema`, um die [Modding](/wiki/Modding)-Seite der Beatmap einzusehen.
+- Drücke `3` oder `Esc` oder klicke auf die Fläche `Abbrechen`, um zur Songauswahl zurückzukehren.
+
+### Beatmap-Karussell
 
-**Ranking Tabelle:**
+![](img/beatmap-cards.jpg)
 
-| Name | Verwendung |
-| :-- | :-- |
-| Lokale Rangliste | Zeigt die lokale Rangliste an. Scores aus dem Multiplayer werden nicht gespeichert. Mit das Drücken der "F2" Taste lässt sich eine verpackte Version des Replays erstellen. Kann möglicherweise in bestimmten Fällen nicht funktionieren. |
-| Nationale Rangliste | **osu!supporter erforderlich!** Zeigt ihren Rang in der nationalen Rangliste an (z. B. Deutschland). |
-| Globale Rangliste | Zeigt die globale Rangliste an. Hier werden die Top 50 Spieler der Map angezeigt, zusammen mit deren Ergebnisanzeige und einem vorhandenen Replay. Durch das Ansehen deren Replays wird es beim Spieler auf dessen Profilseite bei "Historical#Replays Watched History" mit aufgelistet. |
-| Globale Rangliste (Aktive Mods) | **osu!supporter erforderlich!** Zeigt die globale Rangliste mit Spielern, die den selben aktiven Mod/Mods verwendet haben, an. |
-| Freundesrangliste | **osu!supporter erforderlich!** Zeigt eine Rangliste mit allen ihren Freunden an. Funktioniert jedoch nicht bei [BanchoBot](/wiki/BanchoBot) und einer leeren Freundesliste. |
+Das Beatmap-Karussell zeigt alle verfügbaren Beatmaps an. Unterschiedliche Beatmaps können unterschiedlich farbige Boxen haben:
 
-Klicke auf das Sprechblasen-Icon, um zum **Schnellzugang zu dieser Map im Web** im Songauswahl zu gelangen:
+| Farbe der Box | Beschreibung |
+| :-: | :-- |
+| **Pink** | Diese Beatmap wurde bis jetzt noch nicht gespielt. |
+| **Orange** | Mindestens ein Schwierigkeitsgrad der Beatmap wurde abgeschlossen. |
+| **Hellblau** | Andere Schwierigkeitsgrade in derselben Beatmap werden angezeigt, sobald eine Beatmap erweitert wurde. |
```

Comment:
Does there really need to be a comma? If you phrase it like `Es werden andere Schwierigkeitsgrade in derselben Beatmap angezeigt, sobald eine Beatmap erweitert wurde.`, there is only one comma.

Reply:
It's a weird one, cause i thought it is referring to the light blue cards being shown when selecting a beatmap. In that case it would probably have to be `wird angezeigt` anyway. :thinking: 

Can just leave it without without the comma, its fine either way.

Reply:
It indeed is refering to these blue cards seen in the picture above (beatmap-cards.jpg). I'm not sure how to express it differently.

I think I get what you mean now. Is it okay to change it to

```suggestion
| **Hellblau** | Andere Schwierigkeitsgrade in derselben Beatmap, die angezeigt werden, sobald eine Beatmap erweitert wurde. |
```

Reply:
Yeah that works